eessi/testsuite/tests/apps/PyTorch/PyTorch_torchvision.py

```diff
@@ -2,79 +2,41 @@
 
 import reframe as rfm
 import reframe.utility.sanity as sn
-# Added only to make the linter happy
 from reframe.core.builtins import parameter, variable, run_after, sanity_function, performance_function
 
-from eessi.testsuite import hooks
-from eessi.testsuite.constants import SCALES, TAGS, DEVICE_TYPES, COMPUTE_UNIT, CPU, NUMA_NODE, GPU
+from eessi.testsuite.constants import DEVICE_TYPES, COMPUTE_UNIT, CPU, NUMA_NODE, GPU
+from eessi.testsuite.eessi_mixin import EESSI_Mixin
 from eessi.testsuite.utils import find_modules
 
 
-class EESSI_PyTorch_torchvision(rfm.RunOnlyRegressionTest):
+class EESSI_PyTorch_torchvision(rfm.RunOnlyRegressionTest, EESSI_Mixin):
+    descr = 'Benchmark that runs a selected torchvision model on synthetic data'
+
     nn_model = parameter(['vgg16', 'resnet50', 'resnet152', 'densenet121', 'mobilenet_v3_large'])
-    scale = parameter(SCALES.keys())
+    bench_name_ci = 'resnet50'
     parallel_strategy = parameter([None, 'ddp'])
-    compute_device = variable(str)
+    device_type = variable(str)
```

Comment:
I guess we can remove this, no? We also don't set the `compute_unit` as a variable...

I guess I used to have it to enforce setting it in the child class. But the `eessi_mixin` enforces you to set a device type anyway (I think)? so no use enforcing it here anymore.

Reply:
good point, removed.